mocks/routes/enableu-2-smile.js

```diff
@@ -3,7 +3,14 @@ const AVG_THEMAS_RESPONSE = require('../fixtures/avg-themas.json');
 const KLACHTEN_RESPONSE = require('../fixtures/klachten.json');
 const settings = require('../settings');
 
-function getSmileIdentifyingField(fields) {
+function getSmileIdentifyingField(fields, core) {
+  if (!fields?.function) {
```

Comment:
is the function here an array? Maybe check if the first array item is of type function?

Reply:
Ja het is inderdaad een array. Het is data met een beetje vage naamgeving die ik krijg aangeleverd.

Hier bij identiefier kan je zien wat er in moet zitten:
```javascript
module.exports = [
  {
    id: 'post-enableu2smile-klachten',
    url: `${settings.MOCK_BASE_PATH}/smile`,
    method: 'POST',
    variants: [
      {
        id: 'standard',
        type: 'intermediate-api-handler',
        options: {
          klachten: {
            identifier: 'readKlacht',  // <----- Deze
            getFieldWithIdentifier: getSmileIdentifyingField,
            body: KLACHTEN_RESPONSE,
          },
          avg: {
            identifier: 'readAVGverzoek', // <----- Deze
            getFieldWithIdentifier: getSmileIdentifyingField,
            body: AVG_RESPONSE,
          },
          avgThemas: {
            identifier: 'readthemaperavgverzoek', // <----- Deze
            getFieldWithIdentifier: getSmileIdentifyingField,
            body: AVG_THEMAS_RESPONSE,
          },
        },
      },
    ],
  },
];

```